db/migrate/20180807153714_add_conversion_host_table.rb

```diff
@@ -0,0 +1,16 @@
+class AddConversionHostTable < ActiveRecord::Migration[5.0]
+  def change
+    create_table :conversion_hosts do |t|
+      t.string  :name
+      t.string  :address
+      t.string  :type
+      t.string  :resource_type
```

Comment:
`t.references :resource, :type => :bigint, :polymorphic => true`

Reply:
:+1: that's slick